exonum-java-binding/launcher-plugin/exonum_java_runtime_plugin/plugin.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 The Exonum Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Any
+
+from exonum_launcher.runtime import RuntimeSpecLoader
+
+try:
+    from .proto import service_runtime_pb2
+except (ModuleNotFoundError, ImportError):
+    raise RuntimeError("Use proto/generate_proto.sh script to generate protobuf definition")
+
+
+class JavaRuntimeSpecLoader(RuntimeSpecLoader):
```

Comment:
DeploySpec

Reply:
I tend to avoid extremely long names, as the users will need to use them in their config files, so I renamed it to `JavaDeploySpecLoader`